src/app/(site analysis)/sites/[siteId]/reports/[reportId]/edit/edit-sections.tsx

```diff
@@ -113,21 +114,22 @@ function UpdateSiteReportSection({
   });
 
   return (
-    <Card>
+    
+    <Card className="p-4">
```

Comment:
I've made changes to this section in https://github.com/patrickchin/haru2-vercel/pull/55
so don't worry about fixing this part